### PR TITLE
Fix auto trace test

### PR DESCRIPTION
**Pull request by VVeremjova**
Please review @LuxoftAKutsan

### CODE REVIEW

File: src/components/utils/test/auto_trace_test.cc [hunk 1]
```diff
@@ -75,7 +75,8 @@ bool CheckAutoTraceDebugInFile(const std::string& testlog) {
   bool isLogFound = false;
   std::string line;
 
-  std::ifstream file_log("AutoTraceTestLogFile.log");
+  std::ifstream file_log(file_system::CurrentWorkingDirectory() +
+                         "/AutoTraceTestLogFile.log");
```
Comment (anosach-luxoft) [inline src/components/utils/test/auto_trace_test.cc]: @VVeremjova please use const defined above for file name

File: src/components/utils/test/auto_trace_test.cc [hunk 2]
```diff
@@ -75,7 +75,8 @@ bool CheckAutoTraceDebugInFile(const std::string& testlog) {
   bool isLogFound = false;
```
Comment (anosach-luxoft) [inline src/components/utils/test/auto_trace_test.cc]: @VVeremjova I believe you don't need this variable

**Comment by anosach-luxoft** [comment]
@VVeremjova what was the reason of test failure?

**Comment by LuxoftAKutsan** [comment]
Reviewed.  @VVeremjova  good question from @anosach-luxoft

File: src/components/utils/test/auto_trace_test.cc (hunk 2 above)
Comment (VVeremjova) [inline src/components/utils/test/auto_trace_test.cc]: @anosach-luxoft This value actually shows passing or failing test

**Comment by VVeremjova** [comment]
@anosach-luxoft The reason of test failure is not found in log file specific string

**Comment by VVeremjova** [comment]
@anosach-luxoft See 5c865a6

**Pull request merged**